src/Integration.TeamExplorer.UnitTests/Integration.TeamExplorer.UnitTests.csproj

```diff
@@ -31,15 +31,15 @@
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>false</Optimize>
-    <OutputPath>bin\Debug\</OutputPath>
+    <OutputPath>bin\Debug\VS$(VersionSpecificSuffix)\</OutputPath>
```

Comment:
All of the projects that have version-specific references now have version-specific output folders.
This makes things simpler when building multiple version on the same machine - avoids MSBuild/VS getting confused about whether it need to rebuild a project or not.

I tried making all of the assembly names version-specific too, but that cause problems with the `InternalsVisibleTo` attributes, so I undid that change.